src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1115,16 +1094,23 @@ k
             }
```

Comment:
I was wondering that myself. `prop` shows up as a lint error because it's not in the file. I figured it might still be used as some weird super-hyper-global sketchy variable business that we were working on trashing. But I haven't really looked into it. Will drop the requested todo.